STATUS: INLINE FAIL
MODIFIED: com.willwinder.universalgcodesender.GrblControllerTest.rawResponseHandlerWithKnownErrorShouldWriteMessageToConsole
MODIFIED FILE: /home/lichengpeng/test/retest/automated_script/winder/Universal-G-Code-Sender/ugs-core/test/com/willwinder/universalgcodesender/GrblControllerTest.java
CLEANER: com.willwinder.universalgcodesender.i18n.LocalizationTest.loadLocalizationThatDoesNotExistShouldRevertToEnglish
CLEANER FILE: /home/lichengpeng/test/retest/automated_script/winder/Universal-G-Code-Sender/ugs-core/test/com/willwinder/universalgcodesender/i18n/LocalizationTest.java
POLLUTER: com.willwinder.universalgcodesender.i18n.LocalizationTest.loadLocalizationThatDoesExistShouldLoad
POLLUTER FILE: /home/lichengpeng/test/retest/automated_script/winder/Universal-G-Code-Sender/ugs-core/test/com/willwinder/universalgcodesender/i18n/LocalizationTest.java
ORIGINAL CLEANER SIZE: 9
NEW CLEANER SIZE: 1
ELAPSED TIME: 38.604000091552734
==========================
@@ -1271,0 +1271,3 @@
+ {
+     boolean containsAllTranslations = Localization.initialize("fake", "LOCALE");
+ }
